src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -252,10 +258,17 @@ class ActivityFeed extends React.Component<Props, State> {
 
     isCommentFeedItemActive = <T, U: { id: string, replies?: Array<Comment>, type: T }>(item: U): boolean => {
         const { activeFeedEntryId } = this.props;
+        const { focusedItemId } = this.state;
         const { replies } = item;
 
         const isActive = this.isFeedItemActive<T, U>(item);
-        return isActive || (!!replies && replies.some(reply => reply.id === activeFeedEntryId));
+        const isFocused = focusedItemId === item.id;
+
+        if (!focusedItemId) {
```

Comment:
Can we have a quick meeting tomorrow to sync up on the intended experience here?

Reply:
yes, sure
put something in my calendar